packages/automated-dispute/src/services/prophetCodec.ts

```diff
@@ -256,4 +257,26 @@ export class ProphetCodec {
         // TODO: throw ProphetEncodingError
         return index;
     }
+
+    /**
+     * Decodes the BondEscalationStatus enum from the contract.
+     *
+     * @param id - The ID of the request.
+     * @param status - The numeric status from the contract.
+     * @returns The corresponding BondEscalationStatus string.
+     */
+    static decodeBondEscalationStatus(id: Hex, status: number): BondEscalationStatus {
+        switch (status) {
+            case 0:
+                return "Active";
+            case 1:
+                return "Resolved";
+            case 2:
+                return "Escalated";
+            case 3:
+                return "NoResolution";
+            default:
+                throw new ProphetDecodingError(id, toHex(status.toString()));
```

Comment:
This ID is to help us identify the property that was decoded!
```suggestion
                throw new ProphetDecodingError("escalation.status", toHex(status.toString()));
```
You can remove the `id` param from this method too.